### PR TITLE
Fix an issue where attrs would not handle braces in attributes properly

### PR DESCRIPTION
**Pull request by facelessuser**
Related https://github.com/Python-Markdown/markdown/issues/1390

### CODE REVIEW

**Review by iBug — CHANGES_REQUESTED** [review]

File: pymdownx/superfences.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
 RE_NESTED_FENCE_START = re.compile(
     r'''(?x)
     (?P<fence>~{3,}|`{3,})[ \t]*                                                    # Fence opening
-    (?:(\{(?P<attrs>[^\}\n]*)\})?|                                                  # Optional attributes or
+    (?:(\{(?P<attrs>[^\n]*)\})?|                                                  # Optional attributes or
```
[the remark this answers is not in the document]
Comment (facelessuser) [reply pymdownx/superfences.py]: > That's two spaces to be added before the comments

Fair enough

> well I'm never a fan of this kind of alignment

🤷🏻

**Comment by facelessuser** [comment]
@gir-bot lgtm

**Comment by facelessuser** [comment]
@gir-bot lgtm

**Comment by facelessuser** [comment]
Codecov is being weird, but I doubt we've changed coverage.

**Pull request merged**